apps/src/templates/InputPrompt.jsx

```diff
@@ -0,0 +1,56 @@
+'use strict';
```

Comment:
I learned earlier today [from Brad](https://github.com/code-dot-org/code-dot-org/pull/9721#discussion_r72837185) that this is now redundant!